infogami/infobase/client.py

```diff
@@ -682,8 +684,8 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not (self == other)
 
-    def __str__(self): return ""
-    def __repr__(self): return ""
+    def __repr__(self):
+        return "Nothing()"
```

Comment:
@cclauss , @anandology 's [suggestion](https://github.com/internetarchive/infogami/issues/148#issuecomment-801571404)  of `<Nothing>` is better. `Nothing()` is the constructor method. `<Nothing>` is more like the `None` behaviour, but (correctly) suggests it's an object, rather than a builtin.

Reply:
Please read https://docs.python.org/3/reference/datamodel.html#object.__repr__ as suggested in the commit message above.